internal/pkg/service/buffer/storage/statistics/repository/delete.go

```diff
@@ -0,0 +1,58 @@
+package repository
+
+import (
+	"fmt"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/storage/statistics"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/etcdop/op"
+)
+
+// DeleteOp returns an etcd operation to delete all statistics associated with the object key.
+// Statistics for the storage.LevelTarget are not deleted but are rolled up to the parent object.
+// This operation should not be used separately but atomically together with the deletion of the object.
+func (r *Repository) DeleteOp(objectKey fmt.Stringer) *op.AtomicOp {
```

Comment:
`DeleteOp` v sebe spaja uz aj `RollupImportedOnCleanupOp`.

Ide o to, ze po case mazeme z databazy stare `files/slices` aj ich statistiky.

Ak boli data uspesne imporotvane do tabulky `storage.LevelTarget`, tak nechceme statistiky uplne zmazat, ale chceme ich ulozit o uroven vyssie.

Predtym to boli 2 metody, a cele to bolo zlozitejsie ale teraz to funguje ako som pisal vyssie ^^^.